### DISCUSSION TITLE
[Enhancement] [table] el-table custom sorting problem

### DISCUSSION THREAD

**Discussion by Internet-Design-user** [element-plus/element-plus#17250] [Feature Request]
Feature type: **`Enhancement`**

## Details

- Enhanced component name: `table`

## Description

When customizing the sorting of element plus tables, it is not possible to retrieve all columns and modify an order property through ref, just like the elementUI version
My requirement is that when I trigger an event in reverse order, it can become in positive order, and vice versa

### Additional comments

_(empty)_

![1718694379202](https://github.com/element-plus/element-plus/assets/32924507/f9d7d10c-2d47-451b-9701-e22c0a23520a)

My screenshot may not be very accurate, I believe you should be able to understand my needs

**Comment by Internet-Design-user** [comment]
![image](https://github.com/element-plus/element-plus/assets/32924507/51260657-dc6c-461f-9e17-e9fb6fb348e3)

I found columns similar to UI at this location, why are they placed so deep

**Comment by warmthsea** [comment]
I understand what you mean. It seems that you are using a value that is not documented.

**Comment by Internet-Design-user** [comment]
Yes, maybe, but I only found this and using it this way meets my project requirements. so using it this way shouldn't affect anything, right？

**Reply by warmthsea** [reply]
> Yes, maybe, but I only found this and using it this way meets my project requirements. so using it this way shouldn't affect anything, right？

https://github.com/element-plus/element-plus/pull/17252

I will add test cases to this PR later, and you are welcome to test it.
